### PR TITLE
Added logs to processor

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1297

### Change description ###
There is 'PDFs found in ...' log entry marking the end of processing zip file, but there is no log entry marking the beginning of the processing. This PR fills this gap make lease expiration problem better analysable.

Logs added to ZipFileProcessor and UploadEnvelopeDocumentsService

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/UploadEnvelopeDocumentsService.java [hunk 1]
```diff
@@ -88,6 +88,8 @@ private void processEnvelope(CloudBlobContainer blobContainer, Envelope envelope
         CloudBlockBlob blob = null;
         Optional<String> leaseId = Optional.empty();
 
+        log.info("Processing envelope. Container {}, File {}", containerName, zipFileName);
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/UploadEnvelopeDocumentsService.java:91]: We already have processing zip file in the logs 
image

Comment (apereverzin) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/UploadEnvelopeDocumentsService.java:91]: ZipFileProcessor::process which produces this log entry is called from BlobProcessorTask (exactly - from FileContentProcessor) and from UploadEnvelopeDocumentsTask (exactly - from UploadEnvelopeDocumentsService). In the first case there is a log entry from the calling method, in the second case - no, unless this new line is merged. With this new line reading logs and finding exact workflows will be less confusing.

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/UploadEnvelopeDocumentsService.java:91]: should the existing one be deleted then?

Comment (apereverzin) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/UploadEnvelopeDocumentsService.java:91]: No. The existing log is in BlobProcessorTask, the method writing this log entry is called by 2 callers, one of these callers writes a log entry, the other caller does not. This PR is fixing this - both callers will write log entries and thus the whole workflow will be traceable.

**Comment by apereverzin** [comment]
Not necessary.

**Pull request closed without merging**